### PR TITLE
refactor flask app

### PR DESCRIPTION
**Pull request by ttung**
### Description
Break up app.py into the flask app initialization and the routes.  In the long term, we will want to split up the routes into separate python modules and this simplifies that future step.

Depends on #107 

### Test plan
1. pytest
1. run locally
1. run on eb

### CODE REVIEW

**Comment by phoenixAja** [comment]
oh no you beat me, i wanted to do this after this demo deadline.

File: src/py/aspen/flask_app/flask_app.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import os
```
Comment (phoenixAja) [inline src/py/aspen/flask_app/flask_app.py]: can we name this src/py/aspen/app/app ?

File: src/py/aspen/flask_app/routes.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
```
Comment (phoenixAja) [inline src/py/aspen/flask_app/routes.py]: can we rename this src/py/aspen/app/views ?

File: src/py/aspen/flask_app/flask_app.py (hunk 1 above)
Comment (ttung) [reply src/py/aspen/flask_app/flask_app.py]: probably.  we'd need to change our entry point as well.

**Review by phoenixAja — APPROVED** [review]
this looks good to me, just wanted to rename some stuff

File: src/py/aspen/flask_app/flask_app.py (hunk 1 above)
Comment (phoenixAja) [reply src/py/aspen/flask_app/flask_app.py]: i just think the flask part is unnecessary, but if it's too annoying i don't mind waiting until later

Comment (ttung) [reply src/py/aspen/flask_app/flask_app.py]: no problem.

**Comment by ttung** [comment]
ok it's all been renamed.

**Pull request merged**